src/authDataManager/twitter.js

```diff
@@ -12,7 +12,7 @@ function validateAuthData(authData, options) {
   client.auth_token_secret = authData.auth_token_secret;
   
   return client.get("/1.1/account/verify_credentials.json").then((data) => {
-    if (data && data.id == authData.id) {
+    if (data && data.id_str == authData.user_id) {
```

Comment:
See my edits to make it clear. It might as well stay id (hellojs returns user_id)